src/Plugin.php

```diff
@@ -65,6 +65,9 @@ public static function init() {
     // Add teaser image to RSS feeds.
     add_action('rss2_item', __NAMESPACE__ . '\Feed::rss2_item');
 
+    // Fix Facebook embeds to keep within wrapper borders.
+    add_filter('oembed_fetch_url', __CLASS__ . '::oembed_fetch_url', 10, 3);
```

Comment:
Not sure whether that is true.  oEmbed content might actually be executed in the admin as well – e.g., when generating previews of content and possibly also for the rich-text editor preview.   Due to that possibility, the hooks are also registered in the admin.